.travis.yml

```diff
@@ -0,0 +1,10 @@
+sudo: false
+language: ruby
+cache: bundler
+rvm:
+  - 2.3.6
+  - 2.4.3
+  - 2.5.0
+script:
+  - export RAILS_ENV=test
```

Comment:
I think the recommended way to define environment variable on Travis is using the `env:` key. The [Travis' Environment Variables docs](https://docs.travis-ci.com/user/environment-variables/#Defining-public-variables-in-.travis.yml) have more details.